boxsdk/object/item.py

```diff
@@ -356,3 +356,70 @@ def metadata(self, scope='global', template='properties'):
             :class:`Metadata`
         """
         return Metadata(self._session, self, scope, template)
+
+    @api_call
+    def add_to_collection(self, collection):
+        """
+        Add the item to a collection.
+
+        :param collection:
+            The collection to add the item to.
+        :type collection:
+            :class:`Collection`
+        :return:
+            This item.
+        :rtype:
+            :class:`Item`
+        """
+        url = self.get_url()
+        params = {
+            'fields': 'collections'
+        }
+        box_response = self._session.get(url, params=params)
+        response = box_response.json()
+
+        collections = response['collections']
+        collections.append({'id': collection.object_id})
+        data = {
+            'collections': collections
+        }
+        box_response = self._session.put(url, data=json.dumps(data))
```

Comment:
Everything here and below is done automatically for you with `self.update_info()` (inherited from `BaseObject`).